dependency/vault_pki_test.go

```diff
@@ -0,0 +1,184 @@
+// go:build ignore
+
+package dependency
+
+import (
+	"bytes"
+	"encoding/pem"
+	"errors"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/consul-template/renderer"
+	"github.com/hashicorp/vault/api"
+)
+
+func Test_VaultPKI_goodFor(t *testing.T) {
+	_, cert, err := getCert([]byte(testCert))
+	if err != nil {
+		t.Error(err)
+	}
+	dur, ok := goodFor(cert)
+	if ok != false {
```

Comment:
I skipped that here as the other tests hit that path and it would take a decent bit of code to create a cert to test what is already tested via the certs provided by vault.

I'll change the test name and add a comment to make that clearer.